src/test/rpc/AccountInfo_test.cpp

```diff
@@ -508,13 +508,16 @@ class AccountInfo_test : public beast::unit_test::suite
 
         Env env(*this, features);
         Account const alice{"alice"};
-        env.fund(XRP(1000), alice);
+        Account const bob{"bob"};
+        env.fund(XRP(1000), alice, bob);
```

Comment:
is it required to fund `bob`'s account in order to test the account flags? 

Also, shouldn't the `clawback` feature be tested with a custom token (different from XRP) ?

Reply:
Custom is only needed for trustlines, we don’t do that here. This line is to just fund bob so that it becomes a valid account

Reply:
okay. But the `clawback` feature is not applicable for XRP token asset right?
Isn't it better to perform this check on an account with custom tokens, instead of XRP tokens?

It appears that we are only checking if certain flags are set/unset, so I guess it doesn't matter.